tests/data/group_system/group_auditing/group_configure_auditd_data_retention/rule_auditd_audispd_configure_remote_server/remote_server_numeric_address.pass.sh

```diff
@@ -1,6 +0,0 @@
-#!/bin/bash
```

Comment:
Yes, a numeric address is valid in the config file, but the OVAL checks that `remote_server` equals to value specified in the XCCDF value. The OVAL check will not be satisfied by setting `remote_server`  to numeric address, therefore this test would fail.

Reply:
Aha! I see, there is no value setting a numeric address, more over, no Profile selecting it.
Ok, we can live without this test. There is not much difference between dot separated strings and dot separated numbers...